src/Features/Core/Portable/Wrapping/InitializerExpression/InitializerExpressionCodeActionComputer.cs

```diff
@@ -0,0 +1,330 @@
+
+                return GetSmartIndentationAfter(openToken);
+            }
+
+            protected override async Task<ImmutableArray<WrappingGroup>> ComputeWrappingGroupsAsync()
+            {
+                var result = ArrayBuilder<WrappingGroup>.GetInstance();
+                await AddWrappingGroups(result).ConfigureAwait(false);
+                return result.ToImmutableAndFree();
+            }
+
+            private async Task AddWrappingGroups(ArrayBuilder<WrappingGroup> result)
+            {
+                result.Add(await GetWrapEveryGroupAsync().ConfigureAwait(false));
+                result.Add(await GetUnwrapGroupAsync().ConfigureAwait(false));
+                result.Add(await GetWrapLongGroupAsync().ConfigureAwait(false));
+            }
+
+            private async Task<WrappingGroup> GetWrapEveryGroupAsync()
+            {
+                var parentTitle = Wrapper.Wrap_every_item;
+
+                var codeActions = ArrayBuilder<WrapItemsAction>.GetInstance();
+
+                codeActions.Add(await GetWrapEveryNestedCodeActionAsync(
+                    parentTitle, WrappingStyle.WrapFirst_IndentRest).ConfigureAwait(false));
+
+                // See comment in GetWrapLongTopLevelCodeActionAsync for explanation of why we're
+                // not inlinable.
+                return new WrappingGroup(isInlinable: false, codeActions.ToImmutableAndFree());
+            }
+
+            private async Task<WrapItemsAction> GetWrapEveryNestedCodeActionAsync(
+                string parentTitle, WrappingStyle wrappingStyle)
+            {
+                var indentationTrivia = GetIndentationTrivia(wrappingStyle);
+
+                var edits = GetWrapEachEdits(wrappingStyle, indentationTrivia);
+                var title = GetNestedCodeActionTitle(wrappingStyle);
+
+                return await TryCreateCodeActionAsync(edits, parentTitle, title).ConfigureAwait(false);
+            }
+
+            private SyntaxTrivia GetIndentationTrivia(WrappingStyle wrappingStyle)
+            {
+                return wrappingStyle == WrappingStyle.UnwrapFirst_AlignRest
+                    ? _afterOpenTokenIndentationTrivia
+                    : _singleIndentationTrivia;
+            }
+
+            private string GetNestedCodeActionTitle(WrappingStyle wrappingStyle)
+                => wrappingStyle switch
+                {
+                    WrappingStyle.WrapFirst_IndentRest => Wrapper.Indent_all_items,
+                    _ => throw ExceptionUtilities.UnexpectedValue(wrappingStyle),
+                };
+
+            private ImmutableArray<Edit> GetWrapEachEdits(
+                WrappingStyle wrappingStyle, SyntaxTrivia indentationTrivia)
+            {
+                var result = ArrayBuilder<Edit>.GetInstance();
+
+                if (_doWrapInitializerOpenBrace)
+                {
+                    int position = FindEndOfParentPosition();
+
+                    var token = _listSyntax.Parent.FindToken(position);
```

Comment:
isn't this just `_listSyntax.GetFirstToken().GetPreviousToken()`?

Reply:
Yes, it turns out it is. I didn't realize the GetPreviousToken would go outside of its span. That is good to know. Thanks!